src/vonage/__init__.py

```diff
@@ -1,753 +1,753 @@
-                code=response.status_code, host=host
-            )
-
-            # Test for standard error format:
-            try:
-                error_data = response.json()
-                if (
-                    "type" in error_data
-                    and "title" in error_data
-                    and "detail" in error_data
-                ):
-                    message = "{title}: {detail} ({type})".format(
-                        title=error_data["title"],
-                        detail=error_data["detail"],
-                        type=error_data["type"],
-                    )
-            except JSONDecodeError:
-                pass
-            raise ClientError(message)
-        elif 500 <= response.status_code < 600:
-            logger.warning(
-                "Server error: %s %r", response.status_code, response.content
-            )
-            message = "{code} response from {host}".format(
-                code=response.status_code, host=host
-            )
-            raise ServerError(message)
-
-    def _jwt_signed_get(self, request_uri, params=None):
-        uri = "https://{api_host}{request_uri}".format(
-            api_host=self.api_host(), request_uri=request_uri
-        )
-
-        return self.parse(
-            self.api_host(),
-            self.session.get(uri, params=params or {}, headers=self._headers()),
-        )
-
-    def _jwt_signed_post(self, request_uri, params):
-        uri = "https://{api_host}{request_uri}".format(
-            api_host=self.api_host(), request_uri=request_uri
-        )
-
-        return self.parse(
-            self.api_host(), self.session.post(uri, json=params, headers=self._headers())
-        )
-
-    def _jwt_signed_put(self, request_uri, params):
-        uri = "https://{api_host}{request_uri}".format(
-            api_host=self.api_host(), request_uri=request_uri
-        )
-
-        return self.parse(
-            self.api_host(), self.session.put(uri, json=params, headers=self._headers())
-        )
-
-    def _jwt_signed_delete(self, request_uri):
-        uri = "https://{api_host}{request_uri}".format(
-            api_host=self.api_host(), request_uri=request_uri
-        )
-
-        return self.parse(
-            self.api_host(), self.session.delete(uri, headers=self._headers())
-        )
-
-    def _headers(self):
-        token = self.generate_application_jwt()
-        return dict(self.headers, Authorization=b"Bearer " + token)
-
-    def generate_application_jwt(self, when=None):
-        iat = int(when if when is not None else time.time())
-
-        payload = dict(self.auth_params)
-        payload.setdefault("application_id", self.application_id)
-        payload.setdefault("iat", iat)
-        payload.setdefault("exp", iat + 60)
-        payload.setdefault("jti", str(uuid4()))
-
-        return jwt.encode(payload, self.private_key, algorithm="RS256")
+from ._internal import ApplicationV2, BasicAuthenticatedServer, _format_date_param

```

Comment:
Another fully changed file

Reply:
Yeah looks like another spacing newline issue.

Reply:
yeah this went from LF -> CR LF